schema/readout/datalinkhandler.jsonnet

```diff
@@ -75,6 +78,16 @@ local datalinkhandler = {
                 doc="Number of seconds to record")
     ], doc="Recording parameters"),
 
+    err_msg : s.record("ErrorMessage", [
+        s.field("apa_number", self.apa_number, 0,
```

Comment:
Where is this used? Also, APA is TPC specific, and the datalinkhandler can read other frontends, not just the TPC. We need to use the GeoID, instead of coming up with APA and Link IDs.